gnmi_server/server.go

```diff
@@ -122,13 +135,17 @@ func NewServer(config *Config, opts []grpc.ServerOption) (*Server, error) {
 		return nil, errors.New("config not provided")
 	}
 
+	workPath := "/etc/sonic/gnmi"
```

Comment:
>/etc [](http://example.com/codeflow?start=14&length=4)

`/etc` is for config files. Working path may fit more under `/var/run`.

Not matter which you choose, define it as shared constant.

Reply:
updated